lib/courrier/envelope.rb

```diff
@@ -35,8 +34,7 @@ def mailer
     end
 
     def recipient_email
-      case recipient
-      when User
+      if is_user?(recipient)
         recipient.email
```

Comment:
That could just be `respond_to? :email` tbh. Doesnt really matter what kind of class it is.